MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -927,6 +929,13 @@ public void changeStatus(final Campaign campaign, final LocalDate today,
 
         switch (status) {
             case ACTIVE:
+                /*
```

Comment:
I assume this is for future development - if so, then we don't need to clean it up.

Reply:
It is, and is only included because I missed this in the second personnel status expansion PR... both of which were split off of the previous death PRs.